ush/templates/FV3LAM_wflow.xml

```diff
@@ -1130,7 +1130,7 @@ MODULES_RUN_TASK_FP script.
              <taskdep task="&RUN_ENKFUPDT_TN;"/>
            {%- endif %}
         </and>
-        {%- else %}
+        {%- elif do_envar_radar_ref %}
```

Comment:
@daviddowellNOAA @chunhuazhou Please review updates here. 
Previously the dependency here does not consider the situation when one wants a hybrid DA with RRFSDAS but doesn't want to do the envar_radar_ref update.

Reply:
This change looks good to me.  Thanks, Guoqing.